downstream/attributes/cost.adoc

```diff
@@ -32,3 +32,5 @@
 :link-azure-export-schedule: link:https://access.redhat.com/documentation/en-us/cost_management_service/2021/html-single/adding_a_microsoft_azure_source_to_cost_management/index#configuring-an-azure-daily-export-schedule_adding-an-azure-source[Configuring a daily Azure data export schedule]
 :link-adding-sources: link:https://access.redhat.com/documentation/en-us/cost_management_service/2021/html-single/getting_started_with_cost_management/index#adding-sources_configuring[Adding sources to cost management]
 :link-console-rbac: link:https://access.redhat.com/documentation/en-us/red_hat_hybrid_cloud_console/2021/html-single/user_access_configuration_guide_for_role-based_access_control_rbac/index[User Access Configuration Guide For Role-Based Access Control (RBAC)]
+:link-restricted: link:https://access.redhat.com/documentation/en-us/cost_management_service/2021/html-single/adding_an_openshift_container_platform_source_to_cost_management/index#assembly-adding-a-restricted-network-source_adding-an-ocp-source[Adding a restricted network source]
```

Comment:
I think if a customer wants to install 2 operators, 1 regular and 1 in restricted mode, the only sections that are necessarily applicable are:
* `1.9.2. Configuring Cost Operator on a restricted network`
* `1.9.3. Downloading cost reports`
The operator can still be installed just like the regular one, it's just that the configuration steps are different.